crates/air-lib/air-parser/src/parser/validator.rs

```diff
@@ -50,8 +50,12 @@ pub struct VariableValidator<'i> {
     /// due to the way how lalrpop work.
     unresolved_iterables: MultiMap<&'i str, Span>,
 
+    /// Contains all met iterable in call and next, they will be resolved after the whole parsing
+    /// due to the way how lalrpop work.
+    multiple_next_candidates: MultiMap<&'i str, Span>,
+
     /// Contains all names that should be checked that they are not iterators.
-    check_for_non_iterators: Vec<(&'i str, Span)>,
+    not_iterators_candidates: Vec<(&'i str, Span)>,
```

Comment:
maybe `forbidden_iterator_names`?

Reply:
Actually, they become forbidden only after postprocessing whether this field contains names that should be processed.